ui/server/graphql/resolvers/index.ts

```diff
@@ -56,11 +56,14 @@ const isMemberOfGroup = async (parent, { groupId }, { user }) => {
 
 const isCollMember = async (parent, { roundId, bucketId }, { user }) => {
   if (!user) throw new Error("You need to be logged in");
-  const roundMember = await getRoundMember({
-    userId: user.id,
-    roundId,
-    bucketId,
-  });
+  let roundMember;
+  try {
+    roundMember = await getRoundMember({
+      userId: user.id,
+      roundId,
+      bucketId,
+    });
+  } catch (err){}
```

Comment:
Why do we need to catch and discard this?

Reply:
preferrably refactor to not need to ditch the error but otherwise at least add a comment for why we do it

Reply:
The reason for catching the error was because we may want to continue execution even if there is error.

But the error is not ignored. If there is any error, the value of roundMember will be undefined which will be handled in next lines of code.

Reply:
@alinauroz can you take care of this one?

Reply:
> @alinauroz can you take care of this one?

yes

Reply:
done